app/Http/Controllers/UserController.php

```diff
@@ -43,14 +46,24 @@ public function login(LoginRequest $request)
         $user = $request->user();
         $token = $user->createToken('Token Name')->accessToken;
 
-        return response()->json(['token' => $token], 200);
+        return response()->json(['token' => $token]);
     }
 
-    public function update(UpdateUserRequest $request)
+    public function update(UpdateUserRequest $request, $id)
     {
-        $user = $request->user();
-        $result = $this->userService->updateUserEmail($request->email, $user);
-        return response()->json(['message' => 'Your email adress successfully updated, new adress is: ' . $result->email], 200);
+        $requestedUser = User::find($id);
```

Comment:
remove 54
55: $result = $this->userService->updateUserEmail($request->email, User::find($id));